packages/geo-diary-v2/src/components/pages/appointment/mobile-layout.tsx

```diff
@@ -22,17 +23,28 @@ export const MobileLayout: React.FC<MobileLayoutProps> = ({ appointments }) => {
   const location = useLocation()
   const history = useHistory()
   const queryParams = qs.parse(location.search)
+
+  if (queryParams.tab === 'map') {
+    return (
+      <>
+        <Section className="flex-shrink-0" isFlex isFlexColumn hasBackground={false} hasPadding={false}>
```

Comment:
Would be nice if we could extend this feature to `Section` component. E.g. isShrink?